pyatlan/generator/class_generator.py

```diff
@@ -407,13 +407,13 @@ def get_indexes_for_attribute() -> dict[IndexType, str]:
         else:
             base_type = get_base_type()
             # Default index
-            if config:
-                if analyzer := config.get("analyzer"):
-                    if analyzer == "atlan_text_analyzer":
-                        if attr_name.endswith(".stemmed"):
-                            searchable[IndexType.STEMMED] = attr_name
-                        else:
-                            searchable[IndexType.TEXT] = attr_name
+            if config and config.get("analyzer"):
```

Comment:
This code be changed to `if config and (analyzer := config.get("analyzer")):` and the next lined deleted.